### PR TITLE
[Doc] Add python examples

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/loading/Spark-connector-starrocks.md [hunk 1]
```diff
@@ -153,7 +153,9 @@ The following two examples explain how to load data with Spark DataFrames Batch
 
 Construct data in memory and load data into the StarRocks table.
 
-1. Write the following scala code snippet in `spark-shell`:
+1. You can write the spark job using scala or python.
```
Comment (amber-create) [inline docs/loading/Spark-connector-starrocks.md:156]: image
-> Spark Load job

File: docs/loading/Spark-connector-starrocks.md [hunk 2]
```diff
@@ -153,7 +153,9 @@ The following two examples explain how to load data with Spark DataFrames Batch
...
+
+   For scala, run the following codes in `spark-shell`:
```
Comment (amber-create) [inline docs/loading/Spark-connector-starrocks.md:158]: image
-> code snippet

File: docs/loading/Spark-connector-starrocks.md [hunk 3]
```diff
@@ -163,15 +165,42 @@ Construct data in memory and load data into the StarRocks table.
...
+        .option("starrocks.fe.http.url", "127.0.0.1:8030")
+        .option("starrocks.fe.jdbc.url", "jdbc:mysql://127.0.0.1:9030")
         .option("starrocks.table.identifier", "test.score_board")
         .option("starrocks.user", "root")
         .option("starrocks.password", "")
         .mode("append")
         .save()
     ```
 
+For python, run the following codes in `pyspark`:
```
Comment (amber-create) [inline docs/loading/Spark-connector-starrocks.md:177]: image
Indent this part more to be aligned with the above "For scala, run the following codes in `spark-shell`:" part.

File: docs/loading/Spark-connector-starrocks.md [hunk 4]
```diff
@@ -163,15 +165,42 @@ Construct data in memory and load data into the StarRocks table.
...
+        .appName("StarRocks Example") \
+        .getOrCreate()
+
+    # 1. Create a DataFrame from a sequence.
+    data = [(1, "starrocks", 100), (2, "spark", 100)]
+    df = spark.sparkContext.parallelize(data) \
+            .toDF(["id", "name", "score"])
+
+    # 2. Write to starrocks with the format "starrocks",
+    # and replace the options with your own.
```
Comment (amber-create) [inline docs/loading/Spark-connector-starrocks.md:193]: image
write "what", and your own "what"?

File: docs/loading/Spark-connector-starrocks.md [hunk 5]
```diff
@@ -163,15 +165,42 @@ Construct data in memory and load data into the StarRocks table.
     // 2. Write to starrocks with the format "starrocks",
     // and replace the options with your own.
     df.write.format("starrocks")
-        .option("starrocks.fe.http.url", "127.0.0.1:8038")
```
Comment (amber-create) [inline docs/loading/Spark-connector-starrocks.md:166]: image
write "what" and your own "what"?

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by amber-create — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport 3.1 3.0 2.5

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-3.1 branch-3.0 branch-2.5

File: docs/loading/Spark-connector-starrocks.md [hunk 6]
```diff
@@ -153,25 +153,54 @@ The following two examples explain how to load data with Spark DataFrames Batch
...
 
     ```Scala
     // 1. Create a DataFrame from a sequence.
     val data = Seq((1, "starrocks", 100), (2, "spark", 100))
     val df = data.toDF("id", "name", "score")
 
-    // 2. Write to starrocks with the format "starrocks",
-    // and replace the options with your own.
+    // 2. Write to StarRocks by configuring the format as "starrocks" and the following options. 
+    // You need to modify the options according your own environment.
```
Comment (evelynzhaojie) [inline docs/loading/Spark-connector-starrocks.md:166]: according to

File: docs/loading/Spark-connector-starrocks.md [hunk 7]
```diff
@@ -215,10 +246,11 @@ Construct a streaming read of data from a CSV file and load data into the StarRo
             .load("/path/to/csv-data")
         )
 
-    // 2. Write to starrocks with the format "starrocks", and replace the options with your own.
+    // 2. Write to StarRocks by configuring the format as "starrocks" and the following options. 
+    // You need to modify the options according your own environment.
```
Comment (evelynzhaojie) [inline docs/loading/Spark-connector-starrocks.md:250]: according to

**Pull request merged**